samples/README.md

```diff
@@ -6,7 +6,7 @@
 
 [![Open in Cloud Shell][shell_img]][shell_link]
 
-None
```

Comment:
@bcoe sup with that? 🙃

Reply:
fixed by https://github.com/googleapis/synthtool/pull/222 late to this party.